lib/sacastats_web/views/character_view.ex

```diff
@@ -112,15 +120,63 @@ defmodule SacaStatsWeb.CharacterView do
     """
   end
 
-  defp build_event_log_item(assigns, %Events.VehicleDestroy{} = vd, %Session{} = _session) do
+  defp build_event_log_item(assigns, %VehicleDestroy{} = vd, %Session{}, character_map) do
+    character_identifier = get_character_name(assigns, character_map, vd.character_id)
+    attacker_identifier = get_character_name(assigns, character_map, vd.attacker_character_id)
+
     ~H"""
     <li>
-      <%= vd.attacker_character_id %> destroyed <%= vd.character_id %>'s <%= SacaStats.vehicles()[vd.vehicle_id]["name"] %>
+      <%= attacker_identifier %> destroyed <%= character_identifier %>'s <%= SacaStats.vehicles()[vd.vehicle_id]["name"] %>
```

Comment:
Not sure if you want to do anything fun for things like crashing when `attacker_identifier == character_identifier` like
[dndmackey](http://lvh.me:4000/character/dndmackey) destroyed [dndmackey](http://lvh.me:4000/character/dndmackey)'s Liberator with (0) while in a Liberator - 2022-08-29 05:19:09
to
[dndmackey](http://lvh.me:4000/character/dndmackey) crashed while in a Liberator - 2022-08-29 05:19:09
(replacing `destroyed dndmackey's Liberator with (0)` with `crashed`) or something like that

Reply:
Yeah this is a good idea, though I think I'll open a new issue to handle the slightly more niche messages like this since this is a pretty big PR already